boards/arm/disco_l475_iot1/pinmux.c

```diff
@@ -46,12 +46,12 @@ static const struct pin_config pinconf[] = {
 #ifdef CONFIG_PWM_STM32_2
 	{STM32_PIN_PA15, STM32L4X_PINMUX_FUNC_PA15_PWM2_CH1},
 #endif /* CONFIG_PWM_STM32_2 */
-#ifdef CONFIG_USB_STM32
+#ifdef CONFIG_USB_DC_STM32
```

Comment:
Change seems fnie but what worrying me is that I actually tested initial code ok.
I should be able to test it again tomorrow, if this can wait. Even if there is no reason it would work as is.

Reply:
Reason it works "as is" is: at reset, default pinmux setting matches correct USB settings.
Looking into it in mode details

Reply:
after testing reference manual check, it appears that default gpio configuration and pin settings are not fully equivalent, but both works equally. Though, pushing an additional commit to remove PA9 setting which is not defined and not requested.